components/map/InterventionMap.tsx

```diff
@@ -148,11 +231,13 @@ function InterventionMap({
             }}
             title="Modifier moyen"
             onCalloutPress={() => handleFireFighterVehiclePress(mean)}
+            draggable
+            onDragEnd={(event) => handleFireFighterVehicleDragEnd(event, mean)}
           >
             <FireFighterVehicle
               color={getDangerCodeColor(mean.danger_code)}
               name={mean.means.label}
-              dashed={!mean.is_on_site}
+              dashed={!(mean.status === 'on_site')}
```

Comment:
Pourquoi pas plutôt comme ça ?
```suggestion
              dashed={mean.status !== 'on_site'}
```